test/sqlite/test_sqlite_extensions/binding.gyp

```diff
@@ -0,0 +1,10 @@
+{
+  "targets": [
+    {
+      "target_name": "sqlite_extension",
+      "type": "shared_library",
+      "sources": [ "extension.c" ],
+      "include_dirs": [ "../../../deps/sqlite" ]
```

Comment:
Shouldn't we be checking for `--shared-sqlite` here?

Reply:
I'm not sure. We weren't before, but it's possible.

Reply:
In any case, the test is skipped when using shared SQLite, so it's not a concern